openai_vector/challenges/default.json

```
@@ -42,61 +42,56 @@
         "retry-until-success": true,
         "include-in-reporting": false
       }
-    },
+    }
     {# serverless-post-ingest-sleep-marker-start #}{%- if post_ingest_sleep|default(false) -%}
     {
```

```suggestion
    ,{
```

We need a leading comma here, otherwise the json will be invalid if post_ingest_sleep is true